Example/Emission/EigenLikeNavigationController.m

```diff
@@ -20,6 +20,9 @@ - (void)viewDidLoad
   [backButton alignLeadingEdgeWithView:self.view predicate:@"12"];
   [backButton constrainWidth:@"40" height:@"40"];
   _backButton = backButton;
+
+  UIKeyCommand *command = [UIKeyCommand keyCommandWithInput:@" " modifierFlags:0 action:@selector(toggleNav)];
+  [self addKeyCommand: command];
 }
```

Comment:
This listens for the space bar inside the iOS app, and when it is tapped - it will toggle the navigation back button. If you're working in a storybook, you don't need that.

![2017-05-22 16_24_28](https://cloud.githubusercontent.com/assets/49038/26316123/23c96598-3f0b-11e7-9583-68df74ba47e9.gif)